src/TLuaInterpreter.cpp

```diff
@@ -3657,61 +3657,37 @@ int TLuaInterpreter::getMudletInfo(lua_State* L)
-                                                  : ("\"" + encodingName + "\" (\"" + originalEncoding + "\")");
-    };
-    for (const auto& encoding : host.mTelnet.getEncodingsList()) {
-        knownEncodings.append(adjustEncoding(encoding));
-    }
-
-    QString encodingNames{QLatin1String(knownEncodings.join(", "))};
-    encodingNames.append(QLatin1Char('.'));
-    // Have to wrap the above message as it is going to be too wide to fit on
-    // the screen. Split it at comma+space, keeping comma and
-    // replacing space with linefeed:
-    int startLineIndex = 0;
-    const QString needle = QStringLiteral(", ");
-    const int maxLineLength = 79;
-    int endLineIndex = std::min(encodingNames.lastIndexOf(needle,startLineIndex + maxLineLength), maxLineLength);
-    // encodingNames.at(endLineIndex) should be on the last comma position in
-    // what will become the first line
-    encodingNames.replace(endLineIndex, 2, QStringLiteral(",\n"));
-    while (encodingNames.lastIndexOf(needle,startLineIndex + maxLineLength) >= 0) {
-        // There is still a "needle" in the considered part of the haystack!
-        startLineIndex = endLineIndex + 2;
-        endLineIndex = std::min(encodingNames.lastIndexOf(needle,startLineIndex + maxLineLength), startLineIndex + maxLineLength);
-
-        encodingNames.replace(endLineIndex, 2, QStringLiteral(",\n"));
-    }
-
-    QByteArray currentEncoding{host.mTelnet.getEncoding()};
-    if (currentEncoding.isEmpty()) {
-        currentEncoding = "\"ASCII\"";
-    } else {
-        currentEncoding = adjustEncoding(currentEncoding);
-    }
-    QString rawMessage{tr("============================== Mudlet Information ==============================\n"
-                          "--------------------------- Server Codec Information ---------------------------\n"
-                          "Current encoding (with internal name if different): %1\n"
-                          "Available encodings (and internal names):\n"
-                          "%2\n"
-                          "===================================== End ======================================\n",
-                          // Intentional comment to separate arguments
-                          "%1 is the current codec in use; %2 is a comma separated list of encoding names "
-                          "(which are always in Engineering English)")
-                       .arg(QLatin1String(currentEncoding),
-                            encodingNames)};
-    host.mpConsole->print(rawMessage, Qt::white, Qt::black);
+        if (currentEncoding.isEmpty()) {
+            currentEncoding = "\"ASCII\"";
+        } else {
+            currentEncoding = adjustEncoding(currentEncoding);
+        }
+    }
 
-    lua_pushboolean(L, true);
-    return 1;
+
+
+    host.postMessage(QStringLiteral("[ INFO ]  - Current encoding: %1").arg(currentEncoding));
+
+    host.postMessage(QStringLiteral("[ INFO ]  - Available encodings:"));
```

Comment:
Each `postMessage(...)` is a separate message - using one without the `[ xxx ] -` lead-tag was only left in as a fall-back in case of error. Also, theoretically the ~~two~~three part of this message could get split by another from a different source (say as the result of something on a timer). Finally, those are GUI messages so ought to be translatable even if the data that is being displayed is fixed Engineering English.

Reply:
I realised that the text should not be translatable, because this debug output is intended mainly for consumption by ourselves and English is the best way for that.